tests/openvino/test_diffusion.py

```diff
@@ -104,12 +104,12 @@ def test_load_vanilla_model_which_is_not_supported(self):
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     @require_diffusers
     def test_ov_pipeline_class_dispatch(self, model_arch: str):
-        auto_pipeline = self.AUTOMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch])
+        auto_pipeline = DiffusionPipeline.from_pretrained(MODEL_NAMES[model_arch])
```

Comment:
let's simply use an if else here on the sana case, because this test is supposed to check whether the `self.OVMODEL_CLASS` returns the equivalent of what `self.AUTOMODEL_CLASS` returns, but now it's comparing `self.OVMODEL_CLASS` with `DiffusionPipeline`

Reply:
done